### PR TITLE
Support changing time of the day for suite_carla

### PR DESCRIPTION
**Pull request by emailweixu**

### CODE REVIEW

File: alf/environments/suite_carla.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
     cd ..
     ./ImportAssert.sh
     easy_install PythonAPI/carla/dist/carla-0.9.9-py3.7-linux-x86_64.egg
+    pip install networkx==2.2
```
Comment (le-horizon) [inline alf/environments/suite_carla.py:34]: where is this needed?

**Review by le-horizon — COMMENTED** [review]
Almost LG, just one question.

Thanks,
Le

File: alf/environments/suite_carla.py (hunk 1 above)
Comment (emailweixu) [reply alf/environments/suite_carla.py:34]: carla needs it.

File: alf/environments/suite_carla.py [hunk 2]
```diff
@@ -888,6 +904,9 @@ def __init__(self,
         self._num_walkers = num_walkers
         self._percentage_walkers_running = percentage_walkers_running
         self._percentage_walkers_crossing = percentage_walkers_crossing
+        self._day_length = day_length
+        self._time_of_the_day = 0.5 * day_length
```
Comment (hnyu) [inline alf/environments/suite_carla.py:908]: Maybe better called "daytime_length"?

Comment (emailweixu) [reply alf/environments/suite_carla.py:908]: daytime_length is a little ambiguous, as it may mean the contrary of night time.

Comment (hnyu) [reply alf/environments/suite_carla.py:908]: >  as it may mean the contrary of night time.

I think that's what it should mean, no?

Comment (emailweixu) [reply alf/environments/suite_carla.py:908]: day_length means the length of the full cycle of one day.

Comment (hnyu) [reply alf/environments/suite_carla.py:908]: Right, I was talking about the variable name "self._time_of_the_day" which seems the same with "self._day_length". But I think it really means the daytime length

Comment (emailweixu) [reply alf/environments/suite_carla.py:908]: time_of_the_day means the current time. It is initialized at the middle of the day. It's constantly changed by update_time_of_the_day().

**Review by le-horizon — APPROVED** [review]
LG

File: alf/environments/suite_carla.py (hunk 2 above)
Comment (hnyu) [reply alf/environments/suite_carla.py:908]: I see. I had a misunderstanding

**Pull request merged**